e2etest/bgp_test.go

```diff
@@ -1183,8 +1152,85 @@ var _ = ginkgo.Describe("BGP", func() {
 		table.Entry("IPV6", "fc00:f853:0ccd:e799::/116", "ipv6", "fc00:f853:ccd:e800::1/128"),
 	)
 
+	ginkgo.Context("validate FRR running configuration", func() {
+		ginkgo.It("Full BFD profile", func() {
+			bfdProfile := config.BfdProfile{
+				Name:             "fullBFDProfile1",
+				ReceiveInterval:  uint32Ptr(93),
+				TransmitInterval: uint32Ptr(95),
+				EchoInterval:     uint32Ptr(97),
+				EchoMode:         boolPtr(true),
+				PassiveMode:      boolPtr(true),
+				MinimumTTL:       uint32Ptr(253),
+			}
+
+			configData := config.File{
+				Pools: []config.AddressPool{
+					{
+						Name:      "bgp-test",
+						Protocol:  config.BGP,
+						Addresses: []string{v4PoolAddresses},
+					},
+				},
+				Peers:       peersForContainers(frrContainers, "ipv4"),
```

Comment:
Will this work for ipv6? (we can also wait for the ci to tell us)

Reply:
Yes, the test should work because it does not make assertion on BGP connectivity with peers. This way it focus on the configuration rendering and it won't break for any other reasons. As a plus, it should be much quicker